runtime/modules/isp_vcu118.py

```diff
@@ -27,6 +27,8 @@
 
 fpga = "gfe"
 
+reset_address = 0x6fff0000
```

Comment:
Can you please make the reset address a param with an appropriate default value?